cloud-recording/reference/release-notes.md

```diff
@@ -9,8 +9,44 @@ description: >
 
 ## Overview
 
-Agora Cloud Recording is an add-on service to record and save voice calls, video calls, and interactive streaming on your cloud storage. Compared with Agora On-Premise Recording, Agora Cloud Recording is more efficient and convenient as it does not require deploying Linux servers.
+Agora Cloud Recording is an add-on service to record and save voice calls, video calls, and interactive streaming on your cloud storage. 
+Compared with Agora On-Premise Recording, Agora Cloud Recording is more efficient and convenient as it does not require deploying Linux servers.
 
+### Compatibility
+
+Agora Cloud Recording is compatible with the following SDKs:
+
+- Agora SDK for Android, iOS, macOS, and Windows v1.7.0 or later.
```

Comment:
Can you check the supported platforms and make sure this information is there and make a link to it please. Also check for broken links in supported platforms. (I just saw one. )

Reply:
The [Supported Platforms](https://docs.agora.io/en/cloud-recording/reference/supported-platforms) page lists, to quote, "... platforms and products you use to develop apps that interact with Agora SD-RTN™". This, however, it about Agora products being compatible between themselves.